src/assets/translations.data.json

```diff
@@ -7,6 +7,10 @@
     "entry.generic-medium": "VR",
     "entry.gearvr-prefix": "Enter on ",
     "entry.gearvr-medium": "GearVR",
+    "entry.device-prefix": "Enter on ",
```

Comment:
yeah in general basically we have tech debt because the choice of i18n library on my part was probably the wrong choice -- it's cumbersome and limited. so i expect we'll probably just use it when it's low-friction but we'll going to certainly need a concerted effort to re-write everything against the alternative one you mentioned. (still currently assigned to me)